### ISSUE TITLE
[Feature Request] [VSkeletonLoader] Add eager prop like VDialog

### ISSUE THREAD

**Issue by thegoldgoat** [vuetifyjs/vuetify#12987]
### Problem to solve
Pre-Render content on mounted for VSkeletonLoader, as it can already be done with VDialogs

### Proposed solution
Add the same prop VDialog has to VSkeletonLoader

**Comment by FriiZe** [comment]
Hello everyone, I started to develop this issue. But I have one question : How can I test my feature ? I mean, how can I know if the componenent is rendered on mounted. Thanks for awnser !

**Comment by thegoldgoat** [comment]
You can set to the child component `ref=child` and then check `this.child != undefined` in the `mounted` function of the father component

**Comment by FriiZe** [comment]
Thanks for awnswer. A PR should come during the week. At this moment, do not hesitate to give me some improvement ways.

**Comment by FriiZe** [comment]
@thegoldgoat Hi, just another little question. I am still working on this feature and if I understand your previous message, the expected behaviour of `console.log(this.$refs.child != undefined)` is `true` ? That's right ?
Thanks for awnser.

**Comment by FriiZe** [comment]
I don't understand why my `console.log` logged always `true` even if eager prop is set to false. Here is a gist. Is there something wrong ? https://gist.github.com/FriiZe/103742178a7a3c6f591b9cac247e01b0

**Comment by thegoldgoat** [comment]
I guess you have to apply a `:loading` property to the Skeleton Loader (?)

**Comment by FriiZe** [comment]
on my Playground ?

**Comment by thegoldgoat** [comment]
Yes on the playground, you should set `:loading="true"` and check whether the child gets rendered


I then suggest just logging this.$refs.child and see the value

**Comment by FriiZe** [comment]
I tried to give `:loading="true"` prop to `<v-skeleton-loader>` and this time, my log on mounted function of Playground returns always `false` any idea ?

**Comment by FriiZe** [comment]
I tried to give `:loading="true"` prop to `<v-skeleton-loader>` and this time, my log on mounted function of Playground (`console.log(this.$refs.child)`)returns always `undefined` any idea ?

**Comment by thegoldgoat** [comment]
That is because the `eager` does not render the component. Actually I don't does VDialog achieve that functionality

**Comment by thegoldgoat** [comment]
Take a look at this: https://github.com/vuetifyjs/vuetify/blob/98b510b2d68a4b064fa2c4fa65f812a681450e38/packages/vuetify/src/components/VCombobox/__tests__/VCombobox.spec.ts#L54

**Comment by thegoldgoat** [comment]
Unfortunately I am not that knowledgeable about 'low level' Vue internals so I don't really know how to help you more than this

**Comment by FriiZe** [comment]
I dont understand. specs files is just to automate some tests no ? So tests files should does not affect my Playground right ?

**Comment by thegoldgoat** [comment]
Sorry as I said I don't know Vue internals nor Vuetify's code base that much

Let me rephrase this issues:

VSkeletonLoader doesn't mount its child if its loading prop is true when VSkeletonLoader gets mounted

The `eager` prop found on some Vuetify's components such as VDialog override this behavior, so that they mount the child component no matter what, even if it should not be visible e.g. `loading=true` in VSkeletonLoader

**Comment by FriiZe** [comment]
Thanks, it is clearer ! I will try to see which difference exists between my `VSkeletonLoader.ts` and `VDialog.ts`. Thanks for your patience

**Comment by johnleider** [comment]
I'm not sure what's actionable here in the latest versions of the framework. There are only 2 conditions where content is loading, if the default slot has content or if loading is set explicitly to true.

If you have any questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
